scripts/build1.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+# this script builds a go binary after applying openssl fips patches.
+
+set -x
+export GOLANG_VER=1.22.2
+TOP_DIR=/tmp/golang-fips/build1
+[[ -d $TOP_DIR ]] || mkdir -p $TOP_DIR
+cd $TOP_DIR
+[[ -d golang-fips ]] || git clone https://github.com/golang-fips/go.git golang-fips
+cd golang-fips
+./scripts/full-initialize-repo.sh go$GOLANG_VER
+cd go/src
+export GO_BUILDTAGS+="goexperiment.strictfipsruntime,!no_openssl"
```

Comment:
You shouldn't need to explicitly add `!no_openssl`.

Reply:
Fixed